Sources/ComposableArchitecture/UIKit/IfLetUIKit.swift

```diff
@@ -49,32 +49,19 @@ extension Store {
     then unwrap: @escaping (Store<Wrapped, Action>) -> Void,
     else: @escaping () -> Void = {}
   ) -> Cancellable where State == Wrapped? {
-    let elseCancellable =
-      self
-      .publisherScope(
-        state: { state in
-          state
-            .removeDuplicates(by: { ($0 != nil) == ($1 != nil) })
+    return self.state
```

Comment:
Oh nice! So you were able to remove `publisherScope` entirely?

Reply:
That was the plan. Wasn't exactly sure what the unwrapped store's expected behaviour was though. If it looks okay to you though, that works for me.

Reply:
The main place we utilize this is the UIKit part of Tic-Tac-Toe demo, and it seemed to perform well there!

Reply:
Yeah I wasn't sure if the `childStores` from `publisherScope` are meant to receive state updates at the same rate as `publisherScope` receives `childStores`. It that makes sense :P